CONTRIBUTING.md

```diff
@@ -37,19 +37,19 @@ We don't have any frequently asked questions yet.
 
 ### Reporting Bugs
 
-This section guides you through submitting a bug report for {project_module_name}. Following these guidelines helps maintainers and the community understand your report :pencil:, reproduce the behaviour :computer: :computer:, and find related reports :mag_right:.
+This section guides you through submitting a bug report for the project. Following these guidelines helps maintainers and the community understand your report :pencil:, reproduce the behaviour :computer: :computer:, and find related reports :mag_right:.
```

Comment:
It's fairly clear you've done a find replace to switch to `the project`. I think the english can be cleaned up a bit by simply remove it completely in many cases. For example here I think it sounds better just being `This section guides you through submitting a bug report`. Can you look through the changes you've made and see where such simplifications still make sense. Simpler is generally better